ArcGISRuntimeSDKQt_QMLSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.qml

```diff
@@ -367,7 +367,7 @@ Rectangle {
                         deviceObjIds = [];
                         lineObjIds = [];
                         mapView.graphicsOverlays.get(0).graphics.clear();
-                        mapView.map.operationalLayers.forEach(function(layer){
+                        mapView.map.operationalLayers.forEach(layer => {
```

Comment:
for these one-liners we can get rid of curly brace and really simplify `mapView.map.operationalLayers.forEach(layer => layer.clearSelection());`

Reply:
there are other instances throughout where we can simplify a bit more